x-pack/plugins/console_extensions/spec/overrides/xpack.security.put_user.json

```diff
@@ -3,7 +3,7 @@
     "data_autocomplete_rules": {
       "metadata": {},
       "password": "",
-      "fullname": "",
+      "full_name": "",
```

Comment:
Same with this.

Reply:
I guess this comes from another PR I did a while ago: https://github.com/elastic/kibana/pull/37033

Sorry, I am new to this whole Git flow process... I'm not sure why this came together with the modifications I made for this particular issue.

Reply:
No worries :)